server/routes/charts.js

```diff
@@ -46,12 +47,14 @@ router.route('/charts/:chart_id').get(ensureAuthenticated, async (req, res) => {
     const { dataDb } = req.app.locals
     const { chart_id } = req.params
     const { userAccess } = req.session
+    const chartFilters = qs.parse(req.query)
```

Comment:
these are not chart filters but parsed query params.

```suggestion
    const parsedQueryParams = qs.parse(req.query)
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/248/commits/147bf8a9965b532f34bb176e1e651f339c9bf387